bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_batchgenerator.py

```diff
@@ -79,7 +79,7 @@ def get_template(
             name_str = str(name)
             self.given_columns.append(name_str)
             column_type = template_dict[str(x_rdf.GetColumnType(name_str))]
-            template_string += column_type + ","
+            template_string = f"{template_string}{column_type},"
```

Comment:
I wonder what is the reasoning behind this change, as it changes the logic in place. The variable `template_string` was previously declared out of the loop and accumulated at each iteration. Now it's just defined every time anew inside the loop, making its definition at line 70 superfluous. Can you elaborate more here?

Reply:
It works as discussed offline